docs/pages/reference/core/Button.mdx

```diff
@@ -166,6 +208,12 @@ const handleRequest = frames(async (ctx) => {
         >
           View on block explorer
         </Button>,
+        // or using button helper
+        button({
+          action: "link",
+          target: `https://www.onceupon.gg/tx/${ctx.message.transactionId}`,
+          label: "View on block explorer",
+        }),
```

Comment:
We actually have type inference in the button array so the button helper isn't really necessary, right?

Reply:
it isn't necessary, it is just a syntactic sugar because intellisense doesn't work that well to hinting you properties. It works, but the hints are polluted.

It is not required to use `button()` helper.

Reply:
It is just an alternative. So I would keep the helper in place.

Reply:
I will also add an example of plain objects to docs.

Reply:
I added also plain object definition. Maybe it would be better to have code snippets as code groups? https://vocs.dev/docs/markdown#code-groups

Reply:
let's keep it as is and then we can revisit.